File: Backend/security-breach-data2/src/main/java/com/spring/securitybreachdata/entity/Organization.java
package com.spring.securitybreachdata.entity;

import java.io.Serializable;

public class Organization implements Serializable{
	private static final long serialVersionUID = -3371143751583305788L;
	
	private int orgId;
	private String orgName;	
	private String orgIndustry;
    
    public int getOrgId() {
		return orgId;
	}

	public void setOrgId(int orgId) {
		this.orgId = orgId;
	}

	public String getOrgName() {
		return orgName;
	}

	public void setOrgName(String orgName) {
		this.orgName = orgName;
	}

	public String getOrgIndustry() {
		return orgIndustry;
	}

	public void setOrgIndustry(String orgIndustry) {
		this.orgIndustry = orgIndustry;
	}
}

File: Frontend/data-breach-visualization/src/app/model/Organization.ts

export class Organization{
  orgId:number;
  orgName:string;
  orgIndustry: string;
  numIncidents:number;
  numRecordsLost:number;

  constructor(orgId:number, orgName:string, orgIndustry:string, numIncidents:number, numRecordsLost:number) {
    this.orgId = orgId;
    this.orgName = orgName;
    this.orgIndustry = orgIndustry;
    this.numIncidents = numIncidents;
    this.numRecordsLost = numRecordsLost;
  }

}
